attestapis/buf.yaml

```diff
@@ -0,0 +1,10 @@
+version: v1
+name: buf.build/nick-mc/attestapis
```

Comment:
we'll need to decide where we publish to.

This PR shouldn't be merged until this is agreed on

Reply:
updated name